tee-worker/client-api/parachain-api/prepare-build/interfaces/trusted_calls/definitions.ts

```diff
@@ -0,0 +1,94 @@
+export default {
```

Comment:
If you want to keep here calls and getters I suggest naming `trusted_calls` -> `trusted_operations`. 

See:

```
        TrustedOperation: {
            _enum: {
                indirect_call: "(TrustedCallSigned)",
                direct_call: "(TrustedCallSigned)",
                get: "(Getter)",
            },
        },
```